studio-client/apps/main/content-hub-adapter-rss/package.json

```diff
@@ -5,18 +5,18 @@
   "license": "CoreMedia Open Source License",
   "private": true,
   "dependencies": {
-    "@coremedia/studio-client.core-icons": "^2110.2.0",
-    "@coremedia/studio-client.main.content-hub-editor-components": "^2110.2.0",
-    "@coremedia/studio-client.main.editor-components": "^2110.2.0",
-    "@jangaroo/runtime": "^1.0.0"
+    "@coremedia/studio-client.core-icons": "^2110.3.0",
```

Comment:
why going to .3 here? Java dependencies stayed on 2, so I would also stay there for studio-client?

Reply:
yeah, but why not going to the freshest point release? that's what idea proposed

Reply:
I would update to .3 in java too.